tests/integration/security/reachability/manual-mtls/reachability_test.go

```diff
@@ -0,0 +1,139 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package reachability
```

Comment:
Conventionally we use the package name as last element in the path. https://blog.golang.org/package-names

You can create file `manual_tls_test.go` under reachability/